src/main/java/org/apache/maven/plugins/changes/ChangesReport.java

```diff
@@ -75,6 +75,14 @@ public class ChangesReport extends AbstractChangesReport {
     @Parameter(property = "changes.addActionDate", defaultValue = "false")
     private boolean addActionDate;
 
+    /**
+     * Whether the changelog action text should be passed as raw text to the Doxia sink or not.
+     *
+     * @since 3.0
+     */
+    @Parameter
+    private boolean passRawText;
```

Comment:
Is passRawText the best name for this parameter? It doesn't feel self-evident to me. Perhaps we should go the other way. I.e. escapeText with a default value of true. I'm not sure.

Reply:
naming is hard!
i'm fine with the current parameter name as it is in line with the doxia API semantics, but if you come up with a better one, i will update the PR accordingly.

Reply:
i'm also fine renaming (and inverting the meaning) it to `escapeText` - please make a decision and i'm happy to change it.

Reply:
I prefer escapeText. I'm not sure if anyone else feels strongly about this.

Reply:
i've renamed the parameter and impl to "escapeText"